test/integration/unregistered_user_can_add_items_to_cart_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class UnregisteredUserCanAddItemsToCartTest < ActionDispatch::IntegrationTest
+
+  # Background: Items, and a user that is not logged in
+  test 'unregistered user can add items to cart' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.